src/examples/filter-chip/TableWithGlobalFiltersExample.tsx

```diff
@@ -0,0 +1,679 @@
+										acc.push(range);
+									} else {
+										last.end = Math.max(last.end, range.end);
+									}
+									return acc;
+								}, []);
+
+								return (
+									mergedRanges.reduce((acc, range, index) => {
+										const prevEnd = index === 0 ? 0 : mergedRanges[index - 1].end;
+										return (
+											acc +
+											text.slice(prevEnd, range.start) +
+											'<mark>' + text.slice(range.start, range.end) + '</mark>'
+										);
+									}, "") +
+									text.slice(mergedRanges.length ? mergedRanges[mergedRanges.length - 1].end : 0)
+								);
+							}
+						`}
+          />
+          <CodeSnippet
+            lang="html"
+            tags="angular"
+            allowCopy={true}
+            code={`
+							<goa-container mt="m" mb="none">
+								<goa-form-item
+									label="Table with Global Filters"
+									id="filterChipInput"
+									[error]="inputError"
+								>
+									<goa-block gap="xs" direction="row">
+										<goa-input
+											name="filterChipInput"
+											aria-labelledby="filterChipInput"
+											placeholder="Filter"
+											[value]="inputValue"
+											maxlength="100"
+											leadingicon="filter"
+											size="large"
+											(onChange)="handleInputChange($event)"
+											(keypress)="handleInputKeyPress($event)"
+										></goa-input>
+
+										<goa-button
+											type="primary"
+											(click)="addInputChip()"
+											[disabled]="inputValue === ''"
+											leadingicon="add"
+										>
+											Add
+										</goa-button>
+									</goa-block>
+								</goa-form-item>
+
+								<ng-container *ngIf="typedChips.length > 0">
+									<div>
+										<goa-chip
+											*ngFor="let typedChip of typedChips; let index = index"
+											[content]="typedChip"
+											(click)="removeTypedChip(typedChip)"
+											[error]="errorChips.includes(typedChip)"
+											mt="s"
+											mr="s"
+										></goa-chip>
+									</div>
+								</ng-container>
+
+								<goa-table width="100%">
+									<thead>
+										<tr>
+											<th *ngFor="let header of HEADERS">{{ header }}</th>
+										</tr>
+									</thead>
+									<tbody>
+										<tr *ngFor="let item of dataFiltered">
+											<td>
+												<goa-badge type="information" [content]="item.status"></goa-badge>
+											</td>
+											<td [innerHTML]="highlightText(item.text)"></td>
+											<td [innerHTML]="highlightText(item.number)"></td>
+											<td>
+												<a
+													[href]="item.action.hyperlink"
+													[innerHTML]="highlightText(item.action.text)"
+												></a>
+											</td>
+										</tr>
+									</tbody>
+								</goa-table>
+							</goa-container>
+            `}
+          />
+        </>
+      )}
+      {language === "react" && (
+        <>
+          <CodeSnippet
+            lang="typescript"
+            tags="react"
+            allowCopy={true}
+            code={`
```

Comment:
These code snippets are quite large because they include logic to filter and highlight matching text. Let me know if you think that logic should be moved into a utility file or, if it's important enough to move in to the Table component logic.